pyinteraph/centrality_analysis.py

```diff
@@ -16,26 +16,96 @@ def get_hubs(G, **kwargs):
     hubs = {n : (d if d >= kwargs["hub"] else 0) for n, d in degree_tuple}
     return hubs
 
+def get_connected_components(G, **kwargs):
```

Comment:
Use the function available in `graph_analysis` to calculate the connected components. Remove this one.